metrics/distances.py

```diff
@@ -0,0 +1,86 @@
+from votekit.profile import PreferenceProfile
+import numpy as np
+import ot
+import networkx as nx
+from typing import Union
+
+# TODO: Update em_dist to manually create graph in the function. Graph should be able to handle
+# or incomplete ballost
+
+
+def earth_mover_dist(
+    pp1: PreferenceProfile, pp2: PreferenceProfile, ballot_graph
+) -> int:
+    """
+    Computes the earth mover distance between two elections. \n
+    Assumes both elections share the same candidates
+    """
+    # Solving Earth Mover Distance
+    election_arrays = profilePairs_to_arrays(pp1, pp2)
+    electA_distr = election_arrays[0]
+    electB_distr = election_arrays[1]
+
+    # TODO:create ballot graph:
+
+    # Floyd Warshall Shortest Distance alorithm. Returns a dictionary of shortest path for each node
+    FW_dist_dict = nx.floyd_warshall(ballot_graph)
+    keysList = list(FW_dist_dict.keys())
```

Comment:
can be shortened to `keys_list = sorted(fw_dict.keys())`

Reply:
done